src/services/shared/repository.ts

```diff
@@ -0,0 +1,11 @@
+import { CantConnectToMongoDbError, UnknownRepositoryError } from "@domain/errors"
```

Comment:
Shouldn't this be scoped under `mongoose`? its not a service in itself.

Reply:
Oh yea, the vast majority of our repository implementations are already in there anyway.

There's a few places outside of `mongoose` where we call `import { parseRepositoryError } from "@services/mongoose/utils"`. Should we expose those `utils` at the package level, or is it ok to reach in like this?